js/apps/admin-ui/src/clients/add/SamlSignature.tsx

```diff
@@ -96,6 +101,7 @@ export const SamlSignature = () => {
               value: name,
             }))}
           />
+          {samlEncryption?.toString() === "true" && <SamlEncryption />}
```

Comment:
There should be no need to do optional chaining here, the default value is provided in the `watch()` call, so if the value is not set it should return `"false"`, there is also no way the value can be anything but a string, as that is the only data type supported by attributes (aside from sets of strings). You can likely get away with the following:

```suggestion
          {samlEncryption === "true" && <SamlEncryption />}
```